ui/pages/onboarding-flow/recovery-phrase/recovery-phrase-chips.js

```diff
@@ -127,16 +150,22 @@ export default function RecoveryPhraseChips({
         className="recovery-phrase__secret"
       >
         <Box
+          key="recovery-phrase-chips"
           display={Display.Grid}
           justifyContent={JustifyContent.center}
           alignItems={AlignItems.center}
           gap={2}
           data-testid="recovery-phrase-chips"
-          data-recovery-phrase={secretRecoveryPhrase.join(':')}
-          data-quiz-words={JSON.stringify(quizWords)}
           className={classnames('recovery-phrase__chips', {
             'recovery-phrase__chips--hidden': !phraseRevealed,
           })}
+          // tests use these attributes to verify the recovery phrase and quiz words
+          {...(process.env.IN_TEST
+            ? {}
+            : {
+                'data-recovery-phrase': phrasesToDisplay.join(':'),
+                'data-quiz-words': JSON.stringify(quizWords),
+              })}
```

Comment:
We generally try to minimize how often a seed phrase is put into memory, so i've removed these helper attributes from non-test builds.